src/components/o-modal/index.scss

```diff
@@ -36,18 +37,19 @@
   @include respond-up(md) {
     margin-top: 20vh;
     width: 80vw;
-    height: 60vh;
+    height: auto;
```

Comment:
isn't this the default? Doesn't look like you override anything here.

Reply:
im overriding as i set height 100% on the mobile screen